src/vsr/replica.zig

```diff
@@ -5336,10 +5336,12 @@ pub fn ReplicaType(
 
             self.sync_reclaim_tables();
 
-            // Request outstanding committed headers to advance our op number:
+            // Request outstanding possibly committed headers to advance our op number:
             // This handles the case of an idle cluster, where a backup will not otherwise advance.
             // This is not required for correctness, but for durability.
-            if (self.op < self.op_repair_max()) {
+            if (self.op < self.op_repair_max() or
+                (self.status == .normal and self.op < self.view_headers.array.get(0).op))
+            {
                 assert(!self.solo());
                 assert(self.replica != self.primary_index(self.view));
 
```

Comment:
Can you add `self.view_headers.array.get(0).op` to the log line below? Right now it just has `self.op_repair_max()`.